apollo-router/src/services/snapshots/apollo_router__services__supergraph_service__tests__deferred_fragment_bounds_nullability-2.snap

```diff
@@ -23,24 +23,6 @@ expression: stream.next_response().await.unwrap()
               "suborga",
               0
             ]
-          },
```

Comment:
this is removing the redundant parts of the snapshot, now that error dispatching is fixed